src/lib.rs

```diff
@@ -135,7 +136,9 @@ impl IOSurface {
                                                   mem::transmute(self.as_concrete_TypeRef()),
                                                   0);
 
-            assert_eq!(gl_error, kCGLNoError);
+            if gl_error != kCGLNoErrer {
+                panic!(&*CStr::from_ptr(CGLErrorString(gl_error)).to_string_lossy());
```

Comment:
I wonder if we should keep this change in rust-layers, just to avoid inconveniencing anybody else who might be using this library and not expecting fatal errors at this point.